s3proxy/__init__.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, bucket_name, path, key, secret, host, port):
         #self.app.debug = True
 
         self.status = self.app.route('/__status')(self.status)
-        self.handle = self.app.route('/<path:path>')(self.handle)
+        self.handle = self.app.route('/')(self.app.route('/<path:path>')(self.handle))
```

Comment:
This chaining seems a bit odd, could you split the handling of / to its own line instead for clarity?